glide.yaml

```diff
@@ -2,6 +2,8 @@ package: github.com/openshift/cluster-config-operator
 import:
 - package: github.com/openshift/library-go
   version: master
```

Comment:
release-4.1 i think

Reply:
adding release-4.1 fixed the reason for needing to change the k8s imports above, updated thanks